apps/web/app/features/stacking/start-pooled-stacking/components/preset-pools.tsx

```diff
@@ -29,7 +29,7 @@ export const pools: Record<PoolName, Pool> = {
     }, // pool address is the same as pool contract
     poxContract: 'WrapperFastPool',
     minimumDelegationAmount: 40_000_000,
-    icon: <ImgFillLoader src="icons/fastpool.webp" width="32" fill="black" />,
+    icon: <ImgFillLoader src="/icons/fastpool.webp" width="32" fill="black" />,
```

Comment:
how come this change?

Reply:
bc absolute paths are better in this case? works on various routes